src/Providers/AuthServiceProvider.php

```diff
@@ -84,7 +85,13 @@ public function boot()
         });
 
         Gate::before(function ($user, $ability) {
-            return optional(User::fromUser($user))->isSuper() ? true : null;
+            Permission::boot();
+
+            $isStatamicPermission = Permission::all()->first(fn ($permission) => $permission->value() === $ability);
+
+            if ($isStatamicPermission) {
+                return optional(User::fromUser($user))->isSuper() ? true : null;
+            }
         });
```

Comment:
I think we also discussed somewhere that our super check should probably be moved to `after`.

Otherwise, if someone wanted to override a gate/policy and explicitly deny the user (even if they are a super) they should be able to `return false`.

https://github.com/statamic/cms/issues/8337